Source/JavaScriptCore/API/JSValueRef.h

```diff
@@ -292,6 +302,35 @@ JS_EXPORT JSValueRef JSValueMakeString(JSContextRef ctx, JSStringRef string);
  */
 JS_EXPORT JSValueRef JSValueMakeSymbol(JSContextRef ctx, JSStringRef description) JSC_API_AVAILABLE(macos(10.15), ios(13.0));
 
+/*!
+ @function
+ @abstract            Creates a JavaScript value of the bigint.
+ @param ctx           The execution context to use.
+ @param bigint        The JSBigInt to assign to the newly created JSValue.
+ @result              A JSValue of the bigint type, representing the value of bigint.
+ */
+JS_EXPORT JSValueRef JSValueMakeBigInt(JSContextRef ctx, JSBigIntRef bigint) JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
+
+/*!
+ @function
+ @abstract            Truncates a bigint to an signed 64-bit integer and creates a JavaScript value of the bigint type with that integer.
+ @param ctx           The execution context to use.
+ @param bigint        The JSBigInt to assign to the newly created JSValue.
+ @param exception     A pointer to a JSValueRef in which to store an exception, if any. Pass NULL if you do not care to store an exception.
+ @result              A JSValue of the bigint type, representing the value of bigint. Or NULL if an exception is thrown.
+ */
+JS_EXPORT JSValueRef JSValueMakeBigInt64(JSContextRef ctx, JSBigIntRef bigint, JSValueRef* exception) JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
+
+/*!
+ @function
+ @abstract            Truncates a bigint to an unsigned 64-bit integer and creates a JavaScript value of the bigint type with that integer.
+ @param ctx           The execution context to use.
+ @param bigint        The JSBigInt to assign to the newly created JSValue.
+ @param exception     A pointer to a JSValueRef in which to store an exception, if any. Pass NULL if you do not care to store an exception.
+ @result              A JSValue of the bigint type, representing the value of bigint. Or NULL if an exception is thrown.
+ */
+JS_EXPORT JSValueRef JSValueMakeBigUInt64(JSContextRef ctx, JSBigIntRef bigint, JSValueRef* exception) JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
```

Comment:
Ditto.

Reply:
It's requested in https://bugs.webkit.org/show_bug.cgi?id=250719.